app/services/katello/candlepin/consumer.rb

```diff
@@ -0,0 +1,29 @@
+module Katello
+  module Candlepin
+    class Consumer
+      include LazyAccessor
+
+      attr_accessor :uuid
+
+      def initialize(uuid)
+        self.uuid = uuid
+      end
+
+      def regenerate_identity_certificates
+        Rails.logger.debug "Regenerating consumer identity certificates: #{name}"
+        Resources::Candlepin::Consumer.regenerate_identity_certificates(self.uuid)
+      rescue => e
+        Rails.logger.debug e.backtrace.join("\n\t")
+        raise e
+      end
+
+      def backend_data
+        Resources::Candlepin::Consumer.get(uuid)
```

Comment:
I thought internally, it was better to use `@` because without it the variable could be mistaken for or mistreated as a local variable whereas the `@` enforces that it must be an instance variable.